policyengine_us/variables/gov/states/ny/tax/income/credits/ny_inflation_rebates.py

```diff
@@ -0,0 +1,41 @@
+from policyengine_us.model_api import *
+
+
+class ny_inflation_rebates(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New York 2025 inflation rebates"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://assembly.state.ny.us/leg/?Text=Y&bn=S3009&default_fld=&leg_video=&term=2025"
+    defined_for = StateCode.NY
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.ny.tax.income.credits.ny_inflation_rebates
+
+        # Check if inflation rebates are in effect
+        if not p.in_effect:
+            return 0
```

Comment:
```suggestion
```

Reply:
we will avoid this by adding this to the 2025 list of the `non_refundable.yaml` in the ny parameters